### ISSUE TITLE
Allow spectators to voice chat pre/post game

### ISSUE THREAD

**Issue by CreativeMod1** [ThePandemoniumInstitute/botc-release#112]
It'd be good to allow people to still be able to chat between games without being sat down in a seat, it's very common for players to want to discuss the previous game, this can be frustrating for an ST who are wanting to start setting up the next game but people who are planning on leaving are still staying sat down as they wanna chat. It also sucks when you're a spec and wanna be involved in the conversation and can't cos they're not in the game.
This also helps us for the stream games as sometimes some subs will stick around on the app between games in case they're needed for the next game, it'd be easier to communicate that if they're able to talk.

A good solution for this would be that specs are able to voice chat up until the moment that the tokens get sent out, and then they get silenced (or maybe make it that they can still chat to each other mid game or something, or able to chat to the ST without needing to be a co-ST, tbh idk what the current mid-game support is for specs, I know there's some things they're able to do).
Maybe also having some visual to see what specs are talking, but smaller than the tokens, maybe a spec list at the side where names light up when they're talking, discord style?

Other alternatives, urm idk, chatting on Discord?

Other info:
You're awesome Bra1n!!!

**Comment by bra1n** [comment]
I think I have an idea how to fix this: by creating an extra voice chat channel that is only accessible to STs and spectators. I'll need to think about how to enable mics for spectators then, though. So it's a bit more work...

**Comment by AHallqvist** [comment]
Not sure how doable it would be, but if a "ghost room" is created then being able to join it even as a seated player when between games would enable post-game chatting while the next game is being set up/some players take a short break. Then when tokens are passed out that room would need to throw out seated players.
Alternative would be to have people being able to join such a room whilst "reserving a seat" and ST could then call people to their chairs and close any shared voice-room.

**Comment by bra1n** [comment]
Yeah, I agree that it would make sense to allow players to join the spectator room in between games as well.
